5_databases/week2/create-db-template.sql

```diff
@@ -0,0 +1,16 @@
+-- Active: 1706440038860@@127.0.0.1@3307
+CREATE DATABASE school_db DEFAULT CHARACTER SET = 'utf8mb4';
+
+USE school_db;
+
+CREATE TABLE class (
```

Comment:
this is a nitty picky comment so it's up to you to address. while your queries are correct, it is a convention to define every column on a separate line. it is cleaner and more readable. in real world, this convention is a must.

Reply:
@MahmoudYounes  yes, I totally agree with that:) But, when I save my file in vscode it is automatically  formatting all in this way like you see